tests/test_client.py

```diff
@@ -15,7 +15,7 @@ def place(coordinator):
         spawn.close()
         assert spawn.exitstatus == 0, spawn.before.strip()
 
-    with pexpect.spawn('python -m labgrid.remote.client -p test set-tags board=bar') as spawn:
+    with pexpect.spawn('python -m labgrid.remote.client -p test set-tags board=8bar') as spawn:
```

Comment:
I think we should make it clearer in the PR description that this error happens when the value _starts with_ a number. Personally, I'd try to add a new test case instead of modifying an existing one, but not sure how feasible that would be. Also, `8bar` is so specific that it makes me think if there's a reason for it, so I'd go with something like `123abc` which looks more like it's for testing purposes.